.circleci/config.yml

```diff
@@ -27,6 +27,9 @@ jobs:
       - image: <<pipeline.parameters.ci_builder_image>>
     steps:
       - checkout
+      - run:
+          name: Install pnpm
```

Comment:
This isn't ideal, we don't have a docker image that is great for this, ci-builder is too heavy, but this can work for now